src/labkey/query/Rows.ts

```diff
@@ -198,7 +198,17 @@ export interface ModifyRowsResults {
 
 export interface Command {
     /** Name of the command to be performed. Must be one of "insert", "update", or "delete". */
-    command: CommandType
+    command: CommandType    
+    /**
+     * Name of a schema defined within the current container.
+     * See also: [How To Find schemaName, queryName & viewName](https://www.labkey.org/Documentation/wiki-page.view?name=findNames).
+     */
+    schemaName: string
```

Comment:
nit: We usually alphabetize the entries for the interface.

```ts
export interface Command {
    command: CommandType
    extraContext?: any
    rows: any[]
    queryName: string
    schemaName: string
}
```

Reply:
alright